lib/asciidoctor/inline_parser/asciidoctor_grammar.treetop

```diff
@@ -161,7 +161,7 @@ grammar AsciidoctorGrammar
   end
 
   rule constrained_mark_exception
-    [a-zA-Z]
+    '[\p{Word}&&[^_]]'r
```

Comment:
I don't know if we should care about it now but I believe it will not work in a JavaScript environment (since as far as I remember the generated code will contain `\p{Word}`). Should we introduce the `CC_WORD` variable ?

Reply:
This is too early to be thinking about now. Once we have it right, we can go back and make it compatible. Otherwise, we could really shoot ourselves in the foot by adding unnecessary complexity at this stage.

Reply:
I agree 👍